server/graphql/v2/object/RequiredBankInformation.ts

```diff
@@ -0,0 +1,31 @@
+import { GraphQLNonNull, GraphQLString, GraphQLObjectType } from 'graphql';
+import GraphQLJSON from 'graphql-type-json';
+
+import transferwise from '../../../paymentProviders/transferwise';
+
+const RequiredBankInformation = new GraphQLObjectType({
+  name: 'RequiredBankInformation',
+  description: 'A list of required bank information inputs',
+  fields: {
+    fields: {
```

Comment:
`fields: { fields: {` Is that correct?

Reply:
Ok, got it now. Still look funky when you read it.

Reply:
Renamed to required fields.